core/src/main/java/org/jboss/sbomer/core/features/sbom/utils/CustomInstantDeserializer.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2023 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.sbomer.core.features.sbom.utils;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.TemporalAccessor;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+public class CustomInstantDeserializer extends JsonDeserializer<Instant> {
+
+    /*
+     * This is to convert dates like "2024-10-01 08:03:25 UTC" into a java.time.Instant
+     */
+    @Override
+    public Instant deserialize(JsonParser p, DeserializationContext ctxt) throws IOException, JsonProcessingException {
+        String date = p.getText().trim();
+        // Remove the "UTC" and parse
+        if (date.endsWith(" UTC")) {
```

Comment:
Does this mean that Errata returns dates in different formats?! At least the in the docs they say it's https://en.wikipedia.org/wiki/ISO_8601.

Reply:
Errata is fine, it's UMB messages which are not. The dates are in format "2024-10-07 07:24:42 UTC" which use a space instead of a "T" to separate the date and time, and use "UTC" instead of "Z" to indicate that the time is in UTC.